src/index.js

```diff
@@ -66,6 +66,38 @@ container.addEventListener('click', (event) => {
       }
     });
   });
+
+  // interactive section
+  const updateComponents = (components, index) => {
+    components[index].completed = !components[index].completed;
+    localStorage.setItem('toDoList', JSON.stringify(components));
+  };
+
+  // Checkbox status
+  const checkBoxes = document.querySelectorAll('.check-box');
+  checkBoxes.forEach((checkBox, index) => {
+    checkBox.addEventListener('change', () => {
+      updateComponents(components, index);
+      showComponents(components);
+    });
+  });
+
+  const clearCompletedTasks = () => {
+    components = components.filter((item) => item.completed === false);
+    let i = 1;
+    components.forEach((component) => {
+      component.index = i;
+      i += 1;
+    });
+    localStorage.setItem('toDoList', JSON.stringify(components));
+    showComponents(components);
+  };
+
+  // Clear completed tasks
+  const clearBtn = document.querySelector('.clear');
+  clearBtn.addEventListener('click', () => {
+    clearCompletedTasks();
+  });
```

Comment:
- [ ] Good job your app works as expected 👏  But according to the project requirements, the methods related to the status updates (`completed`: `true` / `false`), should be in a new JavaScript files which should be imported as a module. Please fix it in order to make your project fill the requerierement  and note that as a developer it is very important to always follow the requirements because they establish a foundation for product vision, scope, cost, and schedule and they ultimately must target finished product quality and performance. Please fix it :+1: